GovUkDesignSystem/Attributes/ValidationAttributes/GovUkHasCustomValidator.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace GovUkDesignSystem.Attributes.ValidationAttributes;
+
+[AttributeUsage(AttributeTargets.Property)]
+public class GovUkHasCustomValidator : ValidationAttribute
+{
+    public string CustomerValidatorPropertyName;
+
+    protected override ValidationResult IsValid(object value, ValidationContext validationContext)
+    {
+        var propertyInfo = validationContext.ObjectInstance.GetType().GetProperty(CustomerValidatorPropertyName);
+            
+        if (propertyInfo == null) throw new ArgumentException($"'{CustomerValidatorPropertyName}' must not be null");
+        if (propertyInfo.PropertyType == typeof(bool))
```

Comment:
```suggestion
        if (propertyInfo.PropertyType != typeof(bool))
```
Also wrong way round?